feat: add js

taka0209 opened the pull request:
jsを追加しました。

taka0209 committed:
feat: add js

jsを追加しました。

matsuwork commented on lesson4/js/main.js:
```diff
@@ -0,0 +1,38 @@
+      image.alt = data[i].alt;
+      anchor.appendChild(image);
+      anchor.appendChild(text);
+      image.after(text);
```
すみません。質問なのですが、
textが2回挿入されているように見えるのですがなぜでしょうか？

taka0209 committed:
jsを修正しました。

image.after(text);は不要だったので削除しました。

 Please enter the commit message for your changes. Lines starting

nyattsu72 commented:
画像とCSSののファイルが無いようなので、
コミット時に必要なファイルは全て上げた方がいいと思います。

ローカルに落とした際にファイルがなかったので気になりました。

taka0209 commented:
> 画像とCSSののファイルが無いようなので、
> コミット時に必要なファイルは全て上げた方がいいと思います。
> 
> ローカルに落とした際にファイルがなかったので気になりました。

コメントありがとうございます。
jsは問題ないですかね？

画像などは特に指定がなかったので準備しませんでした。
適当な画像を準備したほうが良かったですね。以後、気をつけます。

nyattsu72 commented:
> > 画像とCSSののファイルが無いようなので、
> > コミット時に必要なファイルは全て上げた方がいいと思います。
> > ローカルに落とした際にファイルがなかったので気になりました。
> 
> コメントありがとうございます。
> jsは問題ないですかね？
> 
> 画像などは特に指定がなかったので準備しませんでした。
> 適当な画像を準備したほうが良かったですね。以後、気をつけます。

jsは問題ないと思います！
個人的にconsoleにエラーが吐き出されていたのが気になったのでコメントさせて頂きました。

taka0209 commented:
> > > 画像とCSSののファイルが無いようなので、
> > > コミット時に必要なファイルは全て上げた方がいいと思います。
> > > ローカルに落とした際にファイルがなかったので気になりました。
> > 
> > 
> > コメントありがとうございます。
> > jsは問題ないですかね？
> > 画像などは特に指定がなかったので準備しませんでした。
> > 適当な画像を準備したほうが良かったですね。以後、気をつけます。
> 
> jsは問題ないと思います！
> 個人的にconsoleにエラーが吐き出されていたのが気になったのでコメントさせて頂きました。

ありがとうございます！
